lib/poda_payload.py

```diff
@@ -80,43 +99,60 @@ def send_blobs(self, syscoind):
             while True:
                 # if prevCL - 7 hours for this block or if no prevCL then 7 hours from tip or if gateway's last block then break
                 if mediantimePrevCl > 0 and (mediantimePrevCl - mediantime) > 7*60*60:
-                    print("Time traversed back over 7 hours from mediantimePrevCl: %d" % mediantimePrevCl)
+                    print(
+                        "Time traversed back over 7 hours from mediantimePrevCl: %d" % mediantimePrevCl)
                     break
                 elif mediantimePrevCl == 0 and (medianTimeTip - mediantime) > 7*60*60:
-                    print("Time traversed back over 7 hours from medianTimeTip: %d" % medianTimeTip)
+                    print(
+                        "Time traversed back over 7 hours from medianTimeTip: %d" % medianTimeTip)
                     break
                 if latestBlock.get('hash') == lastblockhash:
-                    print("Found last block hash during traversal: %s" % lastblockhash)
+                    print("Found last block hash during traversal: %s" %
+                          lastblockhash)
                     break
                 # only process blocks that have not been processed already
                 if self.get_local_block_processed(latestHash) is False:
                     # get txids and check PoDA
                     items = latestBlock.get('tx')
                     for txid in items:
                         try:
-                            blobresponse = syscoind.rpc_command('getnevmblobdata', txid, True)  
+                            blobresponse = syscoind.rpc_command(
+                                'getnevmblobdata', txid, True)
                             try:
-                                print("checking PoDA txid {0} {1}".format(txid, self.bucketname))
-                                self.s3.Object(self.bucketname, blobresponse.get('versionhash')).load()
+                                print("checking PoDA txid {0} {1}".format(
+                                    txid, self.bucketname))
+                                self.s3.Object(self.bucketname, blobresponse.get(
+                                    'versionhash')).load()
                             except botocore.exceptions.ClientError as e:
                                 if e.response['Error']['Code'] == "404":
-                                    print("Found PoDA txid! storing in db: %s" % blobresponse.get('versionhash'))
+                                    print("Found PoDA txid! storing in db: %s" %
```

Comment:
can you guys fix the spaces so only the differences show up.. they are all over the place and make the review needlessly hard

Reply:
or do separate PRs
- real code changes
- code formatting changes

Reply:
@sidhujag,  Thanks for the review, The space and formatting adding weren't intended, It was done by a preset code checker [autopep8](https://pypi.org/project/autopep8/)
we would roll back to fit the initial work and disable any code checker.

Reply:
Thanks, We pulled them all out